tests/multihost/complete/data/plans.fmf

```diff
@@ -21,12 +21,16 @@ prepare:
   - how: shell
     script:
       - for envvar in $(compgen -A variable | egrep 'SERVERS|TMT' | sort); do echo "$envvar=${!envvar}"; done
```

Comment:
The `SERVERS` variable has been removed?

Reply:
Correct, it's been removed. Test no longer looks for it after 2526d401.